pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/TransformFunctionFactory.java

```diff
@@ -58,6 +58,8 @@ private TransformFunctionFactory() {
           put(TimeConversionTransformFunction.FUNCTION_NAME.toLowerCase(), TimeConversionTransformFunction.class);
           put(DateTimeConversionTransformFunction.FUNCTION_NAME.toLowerCase(),
               DateTimeConversionTransformFunction.class);
+          put(PrestoDateTruncFunction.FUNCTION_NAME.toLowerCase(),
```

Comment:
(nit) one line

Reply:
It crosses the line limit and the checked in codestyle mandates two lines here :)